virtcontainers/sandbox.go

```diff
@@ -1569,6 +1569,11 @@ func (s *Sandbox) Stop() error {
 	span, _ := s.trace("stop")
 	defer span.Finish()
 
+	if s.state.State == StateStopped {
+		s.Logger().Info("sandbox already stopped")
+		return nil
```

Comment:
How about adding a new unit tests for this scenario, asserting `nil` is returned when state is stopped?